packages/renderer/src/lib/volume/VolumesList.svelte

```diff
@@ -215,76 +235,14 @@ function gotoCreateVolume(): void {
-              on:click="{event => toggleAllVolumes(event.detail)}" />
-          </th>
-          <th class="text-center font-extrabold w-10 px-2">status</th>
-          <th class="w-10">Name</th>
-          <th class="px-6 whitespace-nowrap">age</th>
-          <th class="px-6 whitespace-nowrap text-end">size</th>
-          <th class="text-right pr-2">Actions</th>
-        </tr>
-      </thead>
-      <tbody>
-        {#each volumes as volume}
-          <tr class="group h-12 bg-charcoal-800 hover:bg-zinc-700">
-            <td class="rounded-tl-lg rounded-bl-lg w-5"> </td>
-            <td class="px-2">
-              <Checkbox
-                title="Toggle volume"
-                bind:checked="{volume.selected}"
-                disabled="{volume.inUse}"
-                disabledTooltip="Volume is used by a container" />
-            </td>
-            <td class="bg-charcoal-800 group-hover:bg-zinc-700 flex flex-row justify-center h-12">
-              <div class="grid place-content-center ml-3 mr-4">
-                <StatusIcon icon="{VolumeIcon}" status="{volume.inUse ? 'USED' : 'UNUSED'}" />
-              </div>
-            </td>
-            <td class="whitespace-nowrap w-10 hover:cursor-pointer" on:click="{() => openDetailsVolume(volume)}">
-              <div class="flex items-center">
-                <div class="">
-                  <div class="flex flex-row items-center">
-                    <div class="text-sm text-gray-300">{volume.shortName}</div>
-                  </div>
-                  <div class="flex flex-row text-xs font-extra-light text-gray-900">
-                    <!-- Hide in case of single engine-->
-                    {#if multipleEngines}
-                      <div class="px-2 inline-flex text-xs font-extralight rounded-full bg-slate-800 text-slate-400">
-                        {volume.engineName}
-                      </div>
-                    {/if}
-                  </div>
-                </div>
-              </div>
-            </td>
-            <td class="px-6 py-2 whitespace-nowrap w-10">
-              <div class="flex items-center">
-                <div class="text-sm text-gray-700">{volume.age}</div>
-              </div>
-            </td>
-            <td class="px-6 py-2 whitespace-nowrap w-10">
-              <div class="flex">
-                <div class="w-full text-right text-sm text-gray-700">{volume.humanSize}</div>
-              </div>
-            </td>
-            <td class="pl-6 text-right whitespace-nowrap rounded-tr-lg rounded-br-lg">
-              <VolumeActions volume="{volume}" />
-            </td>
-          </tr>
-          <tr><td class="leading-[8px]">&nbsp;</td></tr>
-        {/each}
-      </tbody>
-    </table>
+    <Table
+      bind:this="{table}"
+      bind:selectedItemsNumber="{selectedItemsNumber}"
+      objects="{volumes}"
+      columns="{columns}"
+      row="{VolumeRow}"
+      helper="{tableHelper}">
```

Comment:
I'm not sure about the helper name, it looks too generic

Reply:
Not sure what else to call it? 🤷🏻‍♂️ Let me know if you have any suggestions.

Reply:
maybe some dedicated methods instead of a generic one. (one entry per helper method/goal) ?

Reply:
As per other comment, TableHelper is gone in latest commit.